chart/kubeapps/values.yaml

```diff
@@ -60,7 +90,7 @@ apprepository:
   image:
     registry: docker.io
     repository: kubeapps/apprepository-controller
-    tag: latest
+    tag: v1.0.0-beta.2
```

Comment:
the same here, it's needed for this to be `latest`

Reply:
👍